frameworks/kafka/tests/test_kerberos_auth.py

```diff
@@ -122,6 +123,13 @@ def kafka_client(kerberos, kafka_server):
         sdk_marathon.destroy_app(client_id)
 
 
+@pytest.mark.dcos_min_version('1.10')
+@sdk_utils.dcos_ee_only
+@pytest.mark.sanity
+def test_no_vip(kafka_client, kafka_server, kerberos):
+    sdk_networks.get_and_test_endpoints(kafka_server["package_name"], kafka_server["service"]["name"], "broker", 2)
```

Comment:
Looks like this only checks count of endpoints.  Would perhaps be better to check expected information in the endpoints.